modules/external-api/src/main/java/org/opencastproject/external/endpoint/LifeCycleManagementEndpoint.java

```diff
@@ -0,0 +1,591 @@
+import static org.opencastproject.util.DateTimeSupport.toUTC;
+import static org.opencastproject.util.RestUtil.getEndpointUrl;
+import static org.opencastproject.util.doc.rest.RestParameter.Type.INTEGER;
+import static org.opencastproject.util.doc.rest.RestParameter.Type.STRING;
+import static org.opencastproject.util.doc.rest.RestParameter.Type.TEXT;
+
+import org.opencastproject.elasticsearch.index.objects.event.EventSearchQueryField;
+import org.opencastproject.external.common.ApiMediaType;
+import org.opencastproject.external.common.ApiResponses;
+import org.opencastproject.lifecyclemanagement.api.Action;
+import org.opencastproject.lifecyclemanagement.api.LifeCyclePolicy;
+import org.opencastproject.lifecyclemanagement.api.LifeCyclePolicyAccessControlEntry;
+import org.opencastproject.lifecyclemanagement.api.LifeCycleService;
+import org.opencastproject.lifecyclemanagement.api.StartWorkflowParameters;
+import org.opencastproject.lifecyclemanagement.api.TargetType;
+import org.opencastproject.lifecyclemanagement.api.Timing;
+import org.opencastproject.lifecyclemanagement.impl.LifeCyclePolicyAccessControlEntryImpl;
+import org.opencastproject.lifecyclemanagement.impl.LifeCyclePolicyImpl;
+import org.opencastproject.metadata.dublincore.EncodingSchemeUtils;
+import org.opencastproject.rest.RestConstants;
+import org.opencastproject.security.api.UnauthorizedException;
+import org.opencastproject.systems.OpencastConstants;
+import org.opencastproject.util.NotFoundException;
+import org.opencastproject.util.UrlSupport;
+import org.opencastproject.util.data.Option;
+import org.opencastproject.util.data.Tuple;
+import org.opencastproject.util.doc.rest.RestParameter;
+import org.opencastproject.util.doc.rest.RestQuery;
+import org.opencastproject.util.doc.rest.RestResponse;
+import org.opencastproject.util.doc.rest.RestService;
+import org.opencastproject.util.requests.SortCriterion;
+
+import com.entwinemedia.fn.data.json.Field;
+import com.entwinemedia.fn.data.json.JValue;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.reflect.TypeToken;
+
+import org.apache.commons.lang3.StringUtils;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/")
```

Comment:
As the jaxrs PR is merged, you must set the path to the opencast.service.path.

Reply:
Thanks for the notification, should be fixed now.